textile/fabric_printing/doctype/print_order/print_order.py

```diff
@@ -823,7 +823,7 @@ def _start_print_order(self, fabric_transfer_qty, publish_progress=True):
 			sales_order = _make_sales_order(self.name, ignore_permissions=True)
 			sales_order.flags.ignore_version = True
 			sales_order.flags.ignore_feed = True
-			sales_order.flags.ignore_permissions = True
+			sales_order.flags.ignore_permissions = ignore_permissions
```

Comment:
This should always be True. Check permissions separately before creating document